x-pack/legacy/plugins/reporting/server/browsers/safe_child_process.ts

```diff
@@ -29,35 +27,40 @@ export function safeChildProcess(
     share()
   );
 
-  // signals that will be sent to the child process as a result of the main process
-  // being sent these signals, or the exit being triggered
-  const signalForChildProcess$ = Rx.merge(
-    // SIGKILL when this process gets a terminal signal
-    ownTerminateSignal$.pipe(mapTo('SIGKILL')),
+  const ownTerminateMapToKill$ = ownTerminateSignal$.pipe(
```

Comment:
This splits apart the observer's operators, giving them variable names and making them less in-context